app/api/dao/user.py

```diff
@@ -128,8 +131,14 @@ def update_user_profile(user_id, data):
         if 'resume_url' in data and data['resume_url']:
             user.resume_url = data['resume_url']
 
-        if 'photo_url' in data and data['photo_url']:
-            user.photo_url = data['photo_url']
+        try:
+            if 'photo_url' in data and data['photo_url']:
+                image_type = data['photo_url'].split(';')[0].split('/')[1]
+                image_name = ".".join((username, image_type))
+                photo_url = aws_upload.upload_to_aws(data['photo_url'], application.congif["AWS_PROFILE_BUCKET"], image_name)
```

Comment:
```suggestion
                photo_url = aws_upload.upload_to_aws(data['photo_url'], application.config["AWS_PROFILE_BUCKET"], image_name)
```

Reply:
Updated .

Reply:
@CleverFool77 this doesn't appear to be changed. Maybe it happened something when you squashed the commits. Can you please change that.
```suggestion
                photo_url = aws_upload.upload_to_aws(data['photo_url'], application.config["AWS_PROFILE_BUCKET"], image_name)
```